app/Http/Controllers/API/DeleteAccountController.php

```diff
@@ -0,0 +1,323 @@
+                            }
+
+                            $adminVerificationLogs = AdminVerificationLog::where([
+                                'user_id' => $userId,
+                                'photo_id' => $photo->id
+                            ])->count();
+
+                            if ($adminVerificationLogs > 0)
+                            {
+                                AdminVerificationLog::where([
+                                    'user_id' => $userId,
+                                    'photo_id' => $photo->id
+                                ])->delete();
+                            }
+
+                            $annotations = Annotation::where([
+                                'photo_id' => $photo->id
+                            ])->count();
+
+                            if ($annotations > 0)
+                            {
+                                Annotation::where([
+                                    'photo_id' => $photo->id
+                                ])->delete();
+                            }
+
+                            $littercoin = Littercoin::where([
+                                'user_id' => $userId,
+                                'photo_id' => $photo->id
+                            ])->count();
+
+                            if ($littercoin > 0)
+                            {
+                                Littercoin::where([
+                                    'user_id' => $userId,
+                                    'photo_id' => $photo->id
+                                ])->delete();
+                            }
+
+                            $photo->delete();
+                        }
+                        catch (\Exception $e)
+                        {
+                            \Log::info(['DeleteAccountController', $e->getMessage()]);
+
+                            return [
+                                'success' => false,
+                                'msg' => 'problem deleting photo'
+                            ];
+                        }
+                    }
+                }
+            });
+
+        try
+        {
+            $adminVerificationLogs = AdminVerificationLog::where('user_id', $userId)->count();
+
+            if ($adminVerificationLogs > 0)
+            {
+                AdminVerificationLog::where('user_id', $userId)->delete();
+            }
+
+            $cleanupUser = DB::table('cleanup_user')->where('user_id', $userId)->count();
+
+            if ($cleanupUser > 0)
+            {
+                DB::table('cleanup_user')->where('user_id', $userId)->delete();
+            }
+
+            $cleanups = Cleanup::where('user_id', $userId)->count();
+
+            if ($cleanups > 0)
+            {
+                Cleanup::where('user_id', $userId)->delete();
+            }
+
+            $countries = Country::select('id', 'created_by', 'user_id_last_uploaded')
+                ->where('created_by', $userId)
```

Comment:
You need to split this into two queries since a country/state/city might have been created by the user to be deleted, but the user_id_last_uploaded could belong to another user, and we're setting it to null regardless. And we don't need the checks.